aten/src/TH/THBFloat16.h

```diff
@@ -0,0 +1,8 @@
+#ifndef TH_BFloat16_H
```

Comment:
I know this is just following THHalf.h but can this just use `#pragma once`?  I assume we didn't do that originally because we wanted C support, but that doesn't seem valid anymore.